[2022-08-03 17:16:46,892] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:16:37.128263+00:00 [queued]>
[2022-08-03 17:16:47,326] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:16:37.128263+00:00 [queued]>
[2022-08-03 17:16:47,494] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:16:47,518] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-03 17:16:47,530] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-03 17:16:47,641] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): decompress-file> on 2022-08-03 17:16:37.128263+00:00
[2022-08-03 17:16:47,757] {standard_task_runner.py:52} INFO - Started process 3265 to run task
[2022-08-03 17:16:47,811] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ltv-pipeline', 'decompress-file', 'manual__2022-08-03T17:16:37.128263+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/dag-test.py', '--cfg-path', '/tmp/tmpdj218mvb', '--error-file', '/tmp/tmpzz8zgg8z']
[2022-08-03 17:16:47,823] {standard_task_runner.py:80} INFO - Job 94: Subtask decompress-file
[2022-08-03 17:16:48,230] {task_command.py:371} INFO - Running <TaskInstance: ltv-pipeline.decompress-file manual__2022-08-03T17:16:37.128263+00:00 [running]> on host 3b0f05b8acbc
[2022-08-03 17:16:48,547] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ltv-pipeline
AIRFLOW_CTX_TASK_ID=decompress-file
AIRFLOW_CTX_EXECUTION_DATE=2022-08-03T17:16:37.128263+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-03T17:16:37.128263+00:00
[2022-08-03 17:16:48,570] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-03 17:16:48,588] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'gunzip /opt/***/files/vehicle_challenge.csv.gz']
[2022-08-03 17:16:48,744] {subprocess.py:85} INFO - Output:
[2022-08-03 17:16:48,799] {subprocess.py:92} INFO - gzip: /opt/***/files/vehicle_challenge.csv already exists;	not overwritten
[2022-08-03 17:16:48,805] {subprocess.py:96} INFO - Command exited with return code 2
[2022-08-03 17:16:48,843] {taskinstance.py:1909} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 195, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2022-08-03 17:16:48,859] {taskinstance.py:1420} INFO - Marking task as FAILED. dag_id=ltv-pipeline, task_id=decompress-file, execution_date=20220803T171637, start_date=20220803T171646, end_date=20220803T171648
[2022-08-03 17:16:48,882] {standard_task_runner.py:97} ERROR - Failed to execute job 94 for task decompress-file (Bash command failed. The command returned a non-zero exit code 2.; 3265)
[2022-08-03 17:16:48,955] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-08-03 17:16:49,039] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
